x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/patch_rule/route.ts

```diff
@@ -87,11 +96,11 @@ export const patchRuleRoute = (router: SecuritySolutionPluginRouter, ml: SetupPl
             ruleId: params.id,
           });
 
-          const rule = await patchRules({
-            rulesClient,
+          const rule = await rulesManagementClient.patchRule({
             existingRule,
             nextParams: params,
           });
+
           if (rule != null && rule.enabled != null && rule.name != null) {
```

Comment:
This also is always true

Reply:
Same question here, but going to delete this check here as well. Thanks.